web/src/HUD.tsx

```diff
@@ -85,12 +88,21 @@ class HUD extends Component<HudProps, HudState> {
     this.toggleSidebar = this.toggleSidebar.bind(this)
   }
 
+  componentWillMount() {
+    this.unlisten = this.history.listen((location, action) => {
+      let tags: Map<string, string> = new Map()
```

Comment:
is there some reason why you didn't use a normal dictionary?

Reply:
where a normal dictionary is just an object?

Reply:
ya